### PR TITLE
Add hover states to profile cards

### LINKED ISSUE #2425: Add hoverstate to project cards
The new project cards on the profile don't have hover states.

### PR DESCRIPTION
**Pull request by juliendonck**
closes #2425 

Screenshot 2021-09-29 at 13 12 02

### CODE REVIEW

**Comment by brandonhaslegs** [comment]
They should also have the same click state as everything else.

**Review by brandonhaslegs — CHANGES_REQUESTED** [review]
They should also have the same press-down state as everything else.

File: ui/App/ProfileScreen/ProjectCardSquare.svelte [hunk 1]
```diff
@@ -27,6 +27,16 @@
...
   }
+
+  .project-card:hover {
+    background-color: var(--color-foreground-level-1);
+  }
+
+  .project-card:active {
+    transition: transform 0.1s ease-in-out;
+    transform: scale(0.95);
+  }
```
Comment (rudolfs) [inline ui/App/ProfileScreen/ProjectCardSquare.svelte:38]: Could we just use the global `.button-transition` class on the respective element?

**Review by brandonhaslegs — APPROVED** [review]
Looks good. Would be cool if when you clicked to copy the Radicle ID, the whole card didn't animate, but that seems impossible to do?

File: ui/App/ProfileScreen/ProjectCardSquare.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/App/ProfileScreen/ProjectCardSquare.svelte:38]: No, cause it also does the opacity thing, which we don't want here. What I could do is make a transition variable and use it in both places.

Comment (rudolfs) [reply ui/App/ProfileScreen/ProjectCardSquare.svelte:38]: Ok, never mind then.

**Pull request merged**